WTWSMS/decisions/WTWSMS_THIS_IS_SPARTA.txt

```diff
@@ -0,0 +1,398 @@
+decisions = {
+	this_is_sparta = {
+		is_high_prio = yes
+
+		potential = {
+			OR = {
+				culture = tsakonian
+				culture = spartan
+				custom_tooltip = {
+					name = is_greek_and_hellenic
+					hidden_tooltip = {
+						AND = {
+							culture = greek
+							OR = {
+								religion = hellenic_pagan
+								religion = cybelian_pagan
+								religion = neoplatonist
+								religion = mithraic_pagan
+								religion = mastimani
+							}
+						}
+					}
+				}
+			}
+			NOT = { has_global_flag = THIS_IS_SPARTA }
+		}
+
+		allow = {
+			independent = yes
+			is_adult = yes
+			OR = {
+				AND = {
+					prestige = 4300
+					num_of_duke_titles = 3
+				}
+				prestige = 8000
+			}
+
+			completely_controls = c_monemvasia
+			martial = 12
+
+			OR = {
+				culture = spartan
+				culture = tsakonian
+				custom_tooltip = {
+					name = is_greek_and_hellenic
+					hidden_tooltip = {
+						AND = {
+							culture = greek
+							OR = {
+								religion = hellenic_pagan
+								religion = cybelian_pagan
+								religion = neoplatonist
+								religion = mithraic_pagan
+								religion = mastimani
+							}
+						}
+					}
+				}
+			}
+
+			religion_group = pagan_group
+
+			NOT = { tier = EMPEROR }
+			NOT = { trait = craven }
+		}
+
+		effect = {
+			custom_tooltip = {
+				name = this_is_sparta.8
```

Comment:
The custom tooltip is also an Event Option. We do need a tooltip which reads "THIS IS SPARTA", because I say we need one and it makes sense in the code. People that play this Mod will understand both the reference and the humor, because they actually have a sense of humor.